src/codewhisperer/models/constants.ts

```diff
@@ -52,8 +52,11 @@ export const CodeWhispererConstants = {
     python: 'python',
     javascript: 'javascript',
     typescript: 'typescript',
+    jsx: 'jsx',
+    vscJsx: 'javascriptreact',
```

Comment:
open another PR to improve readability here https://github.com/aws/aws-toolkit-vscode/pull/2877/files

Reply:
will update this part once ^^ PR get merged